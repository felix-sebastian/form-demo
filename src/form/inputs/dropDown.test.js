import React from "react";
import DropDown from "./dropDown";
import { render } from "@testing-library/react";

it("hides the flyout", () => {
  const { baseElement } = render(
    <DropDown pill={<div>foo</div>} flyout={<div>bar</div>} />
  );

  expect(baseElement).toMatchSnapshot();
});

it("displays the flyout", () => {
  const { baseElement } = render(
    <DropDown focused pill={<div>foo</div>} flyout={<div>bar</div>} />
  );

  expect(baseElement).toMatchSnapshot();
});
