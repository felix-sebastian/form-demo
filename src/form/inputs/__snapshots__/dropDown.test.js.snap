// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the flyout 1`] = `
<body>
  <div>
    <div
      class="sc-AykKE kwvMmO"
    >
      <div>
        foo
      </div>
      <div
        class="sc-AykKC hzReHz"
      >
        <div
          class="sc-AykKD gWCpNp"
        >
          <div>
            bar
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;

exports[`hides the flyout 1`] = `
<body>
  <div>
    <div
      class="sc-AykKE kwvMmO"
    >
      <div>
        foo
      </div>
      <div
        class="sc-AykKC hzReHz"
      />
    </div>
  </div>
</body>
`;
